packages/website/src/components/switch-link.tsx

```diff
@@ -93,15 +103,17 @@ const SwitchLink = ({ href, children, router, ...rest }: Props) => {
   }
 
   return (
-    <Link href={newHref}>
-      <a {...rest}>{children}</a>
+    <Link href={newHref} passHref={passHref}>
+      <a {...rest}>{children(false)}</a>
     </Link>
   );
 };
 
 SwitchLink.propTypes = {
   href: PropTypes.string.isRequired,
-  children: PropTypes.node.isRequired,
+  children: PropTypes.func.isRequired,
+  includeShortcutIcon: PropTypes.bool, // for external links, default true
+  passHref: PropTypes.bool, // for next/link
```

Comment:
Do we need to add propTypes since we are already passing in types? Can we simply remove this?

If yes, can you please remove these propTypes in this PR

Reply:
I've removed the propTypes and am hoping that the build success/failure will tell me that these are necessary/not here.